### DISCUSSION TITLE
Does Quality adjust if set to other than auto?

### DISCUSSION THREAD

**Discussion by vidiot1969** [vimeo/player.js#857] [General]
Hi.  I have a 4K video, but do not wish it to be viewed in 4K at the current time.  What I would like to do is set quality=1080p, but what happens if client bandwidth is lacking?   Will the API lower the resolution automatically (like quality=auto), or will it enforce 1080p no matter what, and cause much buffering?

I know I could re-encode, but just looking for an alternative.

Thanks

**Comment by bdougherty** [comment]
The quality parameter specifically locks the player to that quality, which is the same behavior as selecting a quality from the menu within the player. I don't think there has been much interest in a parameter that would set the cap, so it's not likely something to be added soon.
